abel/transform.py

```diff
@@ -101,7 +101,10 @@ class Transform(object):
 
     symmetry_axis : None, int or tuple
         Symmetrize the image about the numpy axis
-        0 (vertical), 1 (horizontal), (0, 1) (both axes)
+        0 (vertical), 1 (horizontal), (0, 1) (both axes). Note that the
+        abel transform is always performed around the vertical axis.
+        This parameters only affects how the image is modified
+        before applying the Abel transform (see below).
```

Comment:
instead of "This parameters only affects how the image is modified before applying the Abel transform (see below).', can we write something like:

"This parameter only affects how the image is modified before (and after) applying the Abel Transform. For more information, see the "Quadrant combining" note below."

Reply:
Sure